docs/dasharo-menu-docs/device-manager.md

```diff
@@ -196,5 +196,3 @@ driver used and link status. Example view with MAC address blurred:
 ![](/images/menus/net_dev_list2.jpeg){ class="center" }
 
 ![](/images/menus/net_dev_list3.jpeg){ class="center" }
-
```

Comment:
Why did we remove that link?

Reply:
Since there is no such image and `mkdocs` gives a warning:
`WARNING  -  [11:12:49] "GET /images/menus/net_dev_list4.jpeg HTTP/1.1" code 404`

Reply:
OK, thx